README.md

```diff
@@ -147,6 +147,7 @@ python/tach/cache/setup.py[L7]: Import 'tach.filesystem.find_project_config_root
 Tach also supports:
 
 - [Manual file configuration](https://docs.gauge.sh/usage/configuration)
+- [Monorepos and namespace packages](https://docs.gauge.sh/usage/configuration#source-roots)
```

Comment:
Small nit - thoughts on the ordering of the list here? Was trying to go from most important to least for the average user, almost put deprecations under inline exceptions. Imo this is prob under strict interfaces/maybe we want to move manual file configuration down too

Reply:
Updated this :+1: